components/omega/doc/design/OmegaV0ShallowWater.md

```diff
@@ -0,0 +1,692 @@
+1. **Omega-2: Eddying ocean with advanced parameterizations.** This model will have sufficient capability to run realistic global simulations, similar to E3SM V1 [Petersen et al. 2019](https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/2018MS001373).
+
+
+We will produce separate design documents for the time-stepping scheme and the tracer equations in Omega-0. Some of the requirements stated here are repeated or implied by the Omega framework design documents, but are included for clarity.
+
+## 2 Requirements
+
+### 2.1  Omega-0 will solve the nonlinear shallow water equations, plus inactive tracers
+
+The governing equations for Omega-0 are the shallow water equations, as described in [Ringler et al. 2010](https://www.sciencedirect.com/science/article/pii/S0021999109006780) eqns 2-7, plus the tracer advection-diffusion equation. These equations are derived from conservation of momentum, volume, and tracers in a single layer. The code will solve a multi-layer formulation with independent, redundant layers in order to test performance with a vertical array dimension. The exact formulation is specified section 3.1 below, with variable definitions in section 3.2.
+
+
+
+Boundary conditions will include both no-slip and free-slip. The original MPAS-Ocean implementation had no-slip boundaries, implemented by setting vorticity to zero at the boundaries. The free-slip implementation is described by Darren Engwirda in a [github discussion](https://github.com/E3SM-Ocean-Discussion/E3SM/pull/49).
+
+### 2.2  Numerical method will be the TRiSK formulation
+
+The horizontal discretization will be taken from
+[Thuburn et al. 2009](https://www.sciencedirect.com/science/article/pii/S0021999109004434) and [Ringler et al. 2010](https://www.sciencedirect.com/science/article/pii/S0021999109006780), as described in the algorithmic formulation in Section 3 below. This is the same base formulation as in MPAS-Ocean. In addition, we will consider small alterations from the original MPAS-Ocean horizontal discretization and include them as options if they are of minimal additional effort and improve the simulated climate. This includes the recent AUST formulation in [Calandrini et al. 2021](https://www.sciencedirect.com/science/article/pii/S146350032100161X) and simple vorticity averaging considered by the Omega team this past year.
+
+### 2.3  Omega-0 will use MPAS format unstructured-mesh domains
+
+We will continue to use the MPAS-format netcdf files for input and output, with the same mesh variable names and dimensions. This will facilitate ease of use and interoperability between MPAS-Ocean and Omega.
```

Comment:
what is mean by 'MPAS-format netcdf file'?  @xylar are MPAS outputs CF compliant now?  I also assume we won't use the same variable names right?

Reply:
This means that we continue to use the same variables in the MPAS mesh specifications, so that MPAS meshes may be used for Omega. 

We are planning to use the same variable names for mesh variables, except that we are changing from camelCase to PascalCase (first letter caps for consistency). We can still read the same files with this case change.

Reply:
Got it, thanks for the explanation.  That sounds good as you describe.